aiohttp/web_urldispatcher.py

```diff
@@ -296,7 +264,11 @@ def get_info(self):
         return {'path': self._path}
 
     def url(self, *, query=None):
-        return self._append_query(self._path, query)
+        super().url()
```

Comment:
Yes, I see that. I mean that simple replace `url(...)` with `url_for` as message says wouldn't be enough for upgrade.